tools/internal_ci/linux/grpc_aws_experiment.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+# This script is responsible for remotely running tests on an ARM instance.
+# It should return a status code useful to the kokoro infrastructure.
+# It currently assumes an instance will be selected by the time this script begins running.
+
+if [ -z "$KOKORO_KEYSTORE_DIR" ]; then
+    echo "KOKORO_KEYSTORE_DIR is unset. This must be run from kokoro"
+    exit 1
+fi
+
+IDENTITY=${KOKORO_KEYSTORE_DIR}/73836_grpc_arm_instance_ssh_private_test_key1
+
+if [ -z "$INSTANCE" ]; then
+    echo "no INSTANCE set, aborting"
+    exit 1
+fi
+
+FILE=grpc_aws_experiment_remote.sh
+chmod 700 $IDENTITY
+ssh -i $IDENTITY -o StrictHostKeyChecking=no ubuntu@$INSTANCE "rm -rf grpc"  # TODO: remove when using fresh instances
+scp -i $IDENTITY -o StrictHostKeyChecking=no -r github/grpc ubuntu@$INSTANCE:
+ssh -i $IDENTITY -o StrictHostKeyChecking=no ubuntu@$INSTANCE "uname -a; ls -l; bash grpc/tools/internal_ci/linux/$FILE" > ssh_log.txt
+cat ssh_log.txt
+# Sync back sponge_log artifacts (wip)
+echo "looking for sponge logs..."
+find . | grep sponge_log
```

Comment:
They are generated by run_tests.py, but there's a cmdline argument to enable the xml reports.

On our CI, this is the command that our continuous C/C++ runs executes:
```
Will run these tests:
  run_tests_c_linux_opt_native: "python tools/run_tests/run_tests.py --use_docker -t -j 16 -x run_tests/c_linux_opt_native/sponge_log.xml --report_suite_name c_linux_opt_native -l c -c opt --iomgr_platform native --bq_result_table aggregate_results --measure_cpu_costs"
  run_tests_c++_linux_opt_native: "python tools/run_tests/run_tests.py --use_docker -t -j 16 -x run_tests/cpp_linux_opt_native/sponge_log.xml --report_suite_name cpp_linux_opt_native -l c++ -c opt --iomgr_platform native --bq_result_table aggregate_results --measure_cpu_costs"
```
(see log https://source.cloud.google.com/results/invocations/22e4ad26-a871-4cc2-a5a5-383cd958812b/log)

The `-x run_tests/c_linux_opt_native/sponge_log.xml` is what enables the sponge_log.xml
The arguments `--use_docker`, `--iomgr_platform native`, `--bq_result_table aggregate_results` and `--measure_cpu_costs`) you can ignore. For `--use_docker` docker you'd need a ARM64 version of the base docker image first ; `--iomgr_platform native` is the default;  `--bq_result_table aggregate_results` wouldn't work since the AWS instance doesn't have credentials to upload to bigquery. 

Note that the bazel run doesn't generate a sponge_log.xml so the best you can do for bazel now is to store the bazel log. (bazel actually can upload results to resultstore via the build events protocol, but you'd need credentials for that and they won't be available on the AWS machine).